### PR TITLE
Init

### PR DESCRIPTION
**Pull request by fgimenez**
Towards https://github.com/giantswarm/giantswarm/issues/1853

Kickoff of the project, only relevant the CI setup

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# How to contribute
+
+shipyard is Apache 2.0 licensed and accepts contributions via GitHub pull requests. This document outlines some of the conventions on commit message formatting, contact points for developers and other resources to make getting your contribution into shipyard easier.
```
Comment (JosephSalisbury) [inline CONTRIBUTING.md]: not `shipyard`, right?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+RUN mkdir -p /opt
+ADD ./e2e-harness /opt/e2e-harness
+
+RUN mkdir -p /opt/resources
+ADD resources/templates/ /opt/resources/templates
+
+RUN \
+    apk -Uuv add --update \
+      build-base \
```
Comment (JosephSalisbury) [inline Dockerfile]: Can we specify particular versions here? Floating versions makes me nervous

File: README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# e2e-harness
+
+[![CircleCI](https://circleci.com/gh/giantswarm/e2e-harness.svg?style=svg)](https://circleci.com/gh/giantswarm/e2e-harness)
```
Comment (JosephSalisbury) [inline README.md]: I know mr timothy prefers `style=shield`, if you wouldn't mind updating

File: cmd/root.go [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var (
+	RootCmd = &cobra.Command{
+		Use:   "e2e-harness",
+		Short: "Harness for custom kubernetes e2e testing",
+		Long:  ``,
```
Comment (JosephSalisbury) [inline cmd/root.go]: Is there a particular reason to not use " here?

Comment (JosephSalisbury) [reply cmd/root.go]: (this is more for my curiosity)

File: CONTRIBUTING.md (hunk 1 above)
Comment (fgimenez) [reply CONTRIBUTING.md]: sure! fixed (there were a couple of them over there..)

File: Dockerfile (hunk 2 above)
Comment (fgimenez) [reply Dockerfile]: Indeed, very good catch, done

File: README.md (hunk 3 above)
Comment (fgimenez) [reply README.md]: sure, done

File: cmd/root.go (hunk 4 above)
Comment (fgimenez) [reply cmd/root.go]: Don't remember where did i got this from, i think that being the Long description backticks make sense if it's going to have more than one line, not the case :) using " now

Comment (fgimenez) [reply cmd/root.go]: Given that it is not being used I've removed the `Long` field

**Comment by fgimenez** [comment]
@JosephSalisbury thx! comments addressed, ready for another pass

**Pull request merged**